butter_cms/tests/test_author.py

```diff
@@ -47,44 +47,44 @@ def test_all_include_recent_posts(self):
         self.assertGreater(len(data), 0)
 
         author_data = data[0]
-        self.assertEqual('John', author_data.get('first_name'))
+        self.assertEqual('SDK', author_data.get('first_name'))
```

Comment:
Could we compare full body, not just single field? For every test for every type of data (authors, category, content_field, etc).

Reply:
Added this in 929c6feb46613cc6663ccf718d7af6c3f74fda52